### PR TITLE
Issues api: changed the descriptions

### PR DESCRIPTION
**Pull request by achebanets**

### CODE REVIEW

File: spec/swagger.yaml [hunk 1]
```diff
@@ -2739,9 +2739,9 @@ paths:
                   issueSeverityLevelCode: LOW
   '/issues-api/v2/projects/{projectId}/issues/{issueUid}/answered':
     put:
-      summary: Change issue attribute "answered"
+      summary: Update the "answered" attribute of an issue
```
Comment (AlexArcher) [inline spec/swagger.yaml]: the issue?

File: spec/swagger.yaml [hunk 2]
```diff
@@ -3236,7 +3236,7 @@ paths:
   '/issues-api/v2/accounts/{accountUid}/projects/{projectId}/issues/{issueUid}/watchers/{watcherUid}':
     put:
       description: >-
-       Associate or assign the watcher to an issue. The watcher will then begin receiving email notifications.
+       Associate or assign the watcher to an issue. The watcher will then begin receiving email notifications.  Emails do not have to have access to the Smartling account to be added as a Watcher. If they have access to the account the emails include links to view the issue in the dashboard.
```
Comment (AlexArcher) [inline spec/swagger.yaml]: an issue -> the issue
the emails -> emails

File: spec/swagger.yaml (hunk 1 above)
Comment (jones-smartling) [reply spec/swagger.yaml]: @hchaps should we quote attributes like this? Just looking for a general practice / guideline?

**Comment by jones-smartling** [comment]
Approved, but lets get @hchaps feedback before merging.

File: spec/swagger.yaml (hunk 1 above)
Comment (hchaps) [reply spec/swagger.yaml]: @jones-smartling : I'd maybe go with something like: 
- Update issue attribute (answered)

And for the other one:
- Update issue attribute (severity level)

**Pull request merged**